src/envo/envo-idranges.owl

```diff
@@ -55,199 +55,199 @@ Datatype: idrange:3
 
     EquivalentTo:
-        xsd:integer[>= 3500001 , <= 3501000]
-        
+        xsd:integer[>= 5000000 , <= 5100000]
+
 Datatype: idrange:17
 
     Annotations:
-        allocatedto: "Anton Van de Putte"
+        allocatedto: "Brandon Whitehead"
 
     EquivalentTo:
-        xsd:integer[>= 3501001 , <= 3501100]
+        xsd:integer[>= 6000000 , <= 6100000]
 
 Datatype: idrange:18
 
     Annotations:
-        allocatedto: "Damion Dooley"
+        allocatedto: "Jimena Linares"
 
     EquivalentTo:
-        xsd:integer[> 3501100 , <= 3509999]
+        xsd:integer[>= 6100001 , <= 6105000]
 
 Datatype: idrange:19
 
     Annotations:
-        allocatedto: "Lauren Chan"
+        allocatedto: "Mei-Chen Liu"
 
     EquivalentTo:
-        xsd:integer[> 3510000 , <= 3519999]
+        xsd:integer[>= 6105001 , <= 6105100]
 
 Datatype: idrange:20
 
     Annotations:
-        allocatedto: "Raissa Meyer"
+        allocatedto: "Isabella Maria Wilkie"
 
     EquivalentTo:
-        xsd:integer[>= 3520000 , <= 3529999]
+        xsd:integer[>= 6105101 , <= 6105200]
 
 Datatype: idrange:21
 
     Annotations:
-        allocatedto: "Brandon Whitehead"
+        allocatedto: "Agronomy Ontology"
 
     EquivalentTo:
-        xsd:integer[>= 6000000 , <= 6100000]
-
+        xsd:integer[>= 6105201 , <= 6105499]
+        
 Datatype: idrange:22
 
     Annotations:
-        allocatedto: "Jimena Linares"
+        allocatedto: "TermGenie"
 
     EquivalentTo:
-        xsd:integer[>= 6100001 , <= 6105000]
+        xsd:integer[>= 8000000 , <= 8999999]
 
 Datatype: idrange:23
 
     Annotations:
-        allocatedto: "Mei-Chen Liu"
+        allocatedto: "Terms generated by design pattern: entity_attribute_location"
 
     EquivalentTo:
-        xsd:integer[>= 6105001 , <= 6105100]
+        xsd:integer[>= 9000000 , <= 9099999]
 
 Datatype: idrange:24
 
     Annotations:
-        allocatedto: "Isabella Maria Wilkie"
+        allocatedto: "Terms generated by design pattern: entity_quality_location"
 
     EquivalentTo:
-        xsd:integer[>= 6105101 , <= 6105200]
+        xsd:integer[>= 9100000 , <= 9199999]
 
 Datatype: idrange:25
 
     Annotations:
-        allocatedto: "National Microbiome Data Collaborative"
+        allocatedto: "Terms generated by design pattern: entity_attributeA"
 
     EquivalentTo:
-        xsd:integer[>= 3600000 , <= 3609999]
-
-Datatype: idrange:26
+        xsd:integer[>= 9200000 , <= 9299999]
+        
+  Datatype: idrange:26
 
     Annotations:
-        allocatedto: "Agronomy Ontology"
+        allocatedto: "ENVO-P, E-Content-E"
 
     EquivalentTo:
-        xsd:integer[>= 6105201 , <= 6105499]
+        xsd:integer[>= 9300000 , <= 9399999]
 
 Datatype: idrange:27
 
     Annotations:
-        allocatedto: "ANZ Soil and Land survey"
+        allocatedto: "ENVO-P, E-NumberOf-E"
 
     EquivalentTo:
-        xsd:integer[>= 5000000 , <= 5100000]
+        xsd:integer[>= 9400000 , <= 9499999]
 
```

Comment:
I can't find the pattern files for these
@cmungall any ideas?

Reply:
hmm, unfortunately it's hard to see the exact changes for this diff due to the git diff algorithm (it is correct, just not the most useful diff)

does it help to go back to a previous version and look at the (poorly named) git "blame", some of these were introduced 6 years ago:

https://github.com/EnvironmentOntology/envo/blame/1281944c3c0baec03cb9bc3ce27ed9d69846df30/src/envo/envo-idranges.owl